File: assignment-1/q2/sanstemp.py
# initial solution
firstNumber = input('Enter First Number ')
secondNumber = input('Enter Second Number ')
#swapping the values by using x, y = y, x
firstNumber, secondNumber = secondNumber, firstNumber
print('firstNumber: ', firstNumber)
print('secondNumber: ', secondNumber)

#alternate solution
firstNumber = input('Enter First Number ')
fl = len(firstNumber)
secondNumber = input('Enter Second Number ')
sl = len(secondNumber)
firstNumber = float(firstNumber) + float(secondNumber)
secondNumber = float(firstNumber) - float(secondNumber)
firstNumber = float(firstNumber) - float(secondNumber)
print('firstNumber: ', str(firstNumber)[:fl])
print('secondNumber: ', str(secondNumber)[:sl])

File: assignment-1/q2/sanstemp.c
#include <stdio.h>
int main()
{
      double firstNumber, secondNumber;

      printf("Enter first number: ");
      scanf("%lf", &firstNumber);

      printf("Enter second number: ");
      scanf("%lf",&secondNumber);

      // write code here that will produce the given output
      firstNumber = firstNumber + secondNumber;
      secondNumber = firstNumber - secondNumber;
      firstNumber = firstNumber - secondNumber;



      printf("\nfirstNumber = %.2lf\n", firstNumber);
      printf("secondNumber = %.2lf\n", secondNumber);

      return 0;
}
